hack/build.sh

```diff
@@ -23,7 +26,7 @@ fi
 VERSION="beta"
 
 # Determine the arch/os combos we're building for
-XC_ARCH=${XC_ARCH:-"386 amd64"}
+XC_ARCH=${XC_ARCH:-"amd64"}
```

Comment:
Can we provide comment that there were some issues with 386? I am not sure of this. What was the reason for excluding 386?

Reply:
main issues with 386 we are not getting livudev-dev package for 386.